.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "TI2806"]
```

Comment:
We probably can't rename their submodule, can we? This name isn't very descriptive. I think, if we leave the link the same, we should be able to edit the name but I'm not sure.